app/posts/manage-teacher-training-applications/2020-05-18-access-control-history.md

```diff
@@ -0,0 +1,551 @@
+
+For a short story of this, see our [Show and Tell slides](https://docs.google.com/presentation/d/12hs4Rqiw2argAST5lCVD7p4t3xTIrDenx8B0ttTQqZs/edit#slide=id.g7714d84945_0_339) 
+
+* [How UCAS works](#how-ucas-works)
+* [Research we've done](#research-we%E2%80%99ve-done)
+* [User needs](#user-needs)
+* [Options analysis session](#options-analysis-session)
+* [Designs, research and iterations](#designs%2C-research-and-iterations)
+* [Policy](#policy)
+* [Next steps](#next-steps)
+* [Current solution](#current-solution)
+* [Worked examples](#worked-examples)
+
+### How UCAS works
+
+* All providers have their own portal to their own applications through UCAS weblink.
+* All accredited bodies of courses have view-only access to applications at a minimum.
+* Some accredited bodies, normally SCITTs, can have access to other provider’s UCAS weblink portals to view and manage all applications. This might occur if the training provider user shares their username or password, or alternatively, the accredited body user is set up as if they were part of the partner organisation.
+* Some School Directs may have access to other School Directs UCAS weblinks within their [alliance] wider organisational partnership.
+
+#### Disadvantages of the current UCAS setup
+
+* Some accredited bodies may have full access to their School Direct providers’ courses. There is a risk these accredited bodies have access to applications which are not ratified by themselves.
+* The UCAS system does not allow for the complex ways providers interact with each other. This may affect notifications, filtering, sharing applications etc.
+* It is not possible to customise permissions to enable different ways of working.
+* Weblink is not  accessible to all users who interact with applications, meaning applications are constantly downloaded and managed outside of the service - this has GDPR implications.
+* There are constraints around the types of sensitive data the system can collect and display due to the permissions setup. For example, detailed information about criminal convictions from the candidate.
+
+__Goal:__ We have an opportunity in Manage to build a more secure and robust access system to mitigate these implications, meet all the user needs around access control and to allow for more sensitive information to be collected and displayed.
+
+We formulated a starting hypothesis to test: 
+> By introducing access controls ( e.g. read only vs. authorised to make decisions) we can better support provider users to manage applications across different organisational structures. Specifically, we can mitigate the risk of information being seen, or decisions being made, by the wrong person.
+>> By implementing this, we’ll be able to give providers more flexibility to manage their own partnerships.
```

Comment:
Hmmm, is this additional indent necessary?

Reply:
Not really no :)